packages/yargs/lib/webpack.js

```diff
@@ -1,11 +1,49 @@
 'use strict';
 
+const { EOL } = require('os');
+
 const prettyMS = require('pretty-ms');
+const prettyBytes = require('pretty-bytes');
+const chalk = require('chalk');
 
 const {
   internal: { BuildError, BuildWarning },
 } = require('@untool/webpack');
 
+const formatAssets = (assets) =>
+  assets
+    .filter(({ name }) => !name.endsWith('.map'))
+    .map(({ name, size, isOverSizeLimit }) => {
+      const prettySize = (() => {
```

Comment:
Why are we using an IIFE here instead of just a nested ternary?
```javascript
const prettySize = isOverSizeLimit
  ? chalk.enabled
    ? chalk.red(prettyBytes(size))
    : `${prettyBytes(size)} !!!`
  : prettyBytes(size);

```

Reply:
I personally dislike nested ternaries - I generally think they are hard to read, but this instance is rather easy on the eye. Will change it accordingly!